### PR TITLE
CASMINST-5735: Update to remove env var for OCI

### PR DESCRIPTION
**Pull request by atifsyedali**
## Summary and Scope

HMS Github workflows now support OCI registries by default, so we don't need to specify the env var for it, which is breaking the linter action currently.

### CODE REVIEW

**Review by ekamin-hpe — APPROVED** [review]
Looks good to me. I do see the linter action is still failing, but further along now.

**Comment by rsjostrand-hpe** [comment]
take a look at this PR for the changes to ct.yaml: https://github.com/Cray-HPE/hms-firmware-action-charts/pull/19/files#diff-a659d2a19976a325533dea0495eed87496438b045319fd292efca590796060e1

**Comment by atifsyedali** [comment]
Thanks @rsjostrand-hpe . Looks like it is good now. FYI @ndavidson-hpe 

Need one more rubber stamp approval before I can merge this in and unblock other PRs.

**Review by leliasen-hpe — APPROVED** [review]

**Pull request merged**
